apps/telegram-bot/core/handlers/buttons.py

```
@@ -57,24 +58,40 @@ async def buttons_process(update: Update, context: CallbackContext) -> None:
-        if data["type"] == "force":
-            metadata_item["message_type"] = MessageType.SHORT
-        await send_item_message(metadata_item, chat_id=chat_id)
-        if data["type"] == "channel":
-            await query.message.reply_text(
-                text=f"Item sent to the channel.",
+
+        if SCRAPE_MODE == "queue":
+            from core import queue_client
+
+            replying_message = await query.message.reply_text(
+                text=f"Item queued for processing...",
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)